draft-lmbdhk-quic-multipath.md

```diff
@@ -655,7 +655,12 @@ ACK_MP frame is formatted as shown in {{fig-mp-ack-format}}.
 
 Compared to the ACK frame specified in {{QUIC-TRANSPORT}}, the following field is added.
 
-Packet Number Space Identifier: An identifier of the path packet number space, which is the sequence number of Destination Connection ID of the 1-RTT packets which are acknowledged by the ACK_MP frame. If the endpoint receives 1-RTT packets with 0-length Connection ID, it SHOULD use Packet Number Space Identifier 0 in ACK_MP frames.
+Packet Number Space Identifier: An identifier of the path packet number space, which is the sequence number of 
+Destination Connection ID of the 1-RTT packets which are acknowledged by the ACK_MP frame. If the endpoint receives 
+1-RTT packets with 0-length Connection ID, it SHOULD use Packet Number Space Identifier 0 in ACK_MP frames. 
+If an endpoint receives a ACK_MP frame with a non-existing packet number space ID, it SHOULD treat this 
```

Comment:
The SHOULD here sounds strange. I would rather set a MUST, otherwise how should we handle ACK_MP of a given Path ID, non-existing at the reception time, but that later exists?